assets/config/postcss.config.js

```diff
@@ -16,6 +16,18 @@ module.exports = {
             ]
         }),
 
+        // PurgeCSS to remove unused classes for better page speed.
+        require("@fullhuman/postcss-purgecss")({
```

Comment:
It might be good to drop a link to the documentation for this config just as a quick reference.